web/index.php

```diff
@@ -63,9 +63,12 @@ function generate_epics_based_on_starting_label($starting_label, $number_of_iter
     $epics = generate_epics_based_on_starting_label($starting_label, $number_of_iterations);
 
     foreach ($epics as $epic) {
+        $iteration_start_date = date('Y-m-d', strtotime('-3 monday', strtotime($epic)));
```

Comment:
Should we add a `TODO` comment here? This is probably pretty specific to our use case.